lib/D2L.Extensibility.AuthSdk.UnitTests/FunctionalApiUriTests.cs

```diff
@@ -288,6 +288,32 @@ public class FunctionalApiUriTests {
 		}
 
 		[Test]
+		public void UserContext_CreateAuthUri_QuerystringIsMaintained() {
+			Uri queryUrl = m_userContext.CreateAuthenticatedUri( TestConstants.API_PATH_WITH_QUERY, "GET" );
+
+			Assert.IsTrue(
+				queryUrl.ToString().Contains( TestConstants.API_PATH_QUERYSTRING ),
+				"The querystring parameter was not present"
+				);
+		}
+
+		[Test]
+		public void UserContext_CreateAuthUri_MultipleQuerystringsAreNotPresent() {
+			const string querystringCharacterEncoded = "%3F";
+
+			Uri queryUrl = m_userContext.CreateAuthenticatedUri( TestConstants.API_PATH_WITH_QUERY, "GET" );
+			bool duplicatedQuery = (

```

Comment:
Hmm... I had added the test in question as specific regression test. Prior to the change to D2LUserContext, both the "?" and "%3F" separator were present in the same URL.

In your opinion, is this a bad test because it's checking for something specific or because it's not clear on what it's testing?